### PR TITLE
S-94173 - jdk17 upgrade

### PR DESCRIPTION
**Pull request by sishwarya**
## Definition of Done

**General**
 - [ ] Branch is up-to-date with master
 - [ ] Code is reviewed and tested by someone in the Kube team
 - [ ] If there are user facing changes (new env variables for example) update the docs and make sure to notify the docs team to update official documentation

### CODE REVIEW

**Review by darkospoljaric — APPROVED** [review]
This PR is okay for xl-release as well. It can be merged into master after both Deploy and Release merge their respective JDk17 branches. If it's merged sooner then the nightly image build will produce a faulty image (JDK17 installed in the OS won't work with a JDK11 distribution out of the box).

**Review by vpugar-digital — APPROVED** [review]

**Pull request merged**
